fe/fe-core/src/main/java/org/apache/doris/analysis/SchemaTableType.java

```diff
@@ -74,7 +74,11 @@ public enum SchemaTableType {
     SCH_ACTIVE_QUERIES("ACTIVE_QUERIES", "ACTIVE_QUERIES", TSchemaTableType.SCH_ACTIVE_QUERIES),
     SCH_WORKLOAD_GROUPS("WORKLOAD_GROUPS", "WORKLOAD_GROUPS", TSchemaTableType.SCH_WORKLOAD_GROUPS),
     SCHE_USER("user", "user", TSchemaTableType.SCH_USER),
-    SCH_PROCS_PRIV("procs_priv", "procs_priv", TSchemaTableType.SCH_PROCS_PRIV);
+    SCH_PROCS_PRIV("procs_priv", "procs_priv", TSchemaTableType.SCH_PROCS_PRIV),
+
+    SCH_WORKLOAD_SCHEDULE_POLICY("WORKLOAD_SCHEDULE_POLICY", "WORKLOAD_SCHEDULE_POLICY",
+            TSchemaTableType.SCH_WORKLOAD_SCHEDULE_POLICY);
+
```

Comment:
And also delete related show proc code, we will not maintain show proc in the future.